.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,55 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.209.6/containers/python-3
```

Comment:
Can the link be to the `main` branch, and not a specific one?
e.g. https://github.com/microsoft/vscode-dev-containers/tree/main/containers/python-3